packages/tokens/_build/style-dictionary.js

```diff
@@ -50,6 +50,34 @@ StyleDictionary.registerTransform({
   },
 });
 
+/**
+ * @function StyleDictionary.registerTransform()
+ * Defines a custom StyleDictionary transform.
+ *
+ * swisspost/px-to-rem:
+ * Used to transform font sizes from px to rem
+ * This ensures accessible font-sizes
+ */
+
+StyleDictionary.registerTransform({
+  name: 'swisspost/px-to-rem',
+  type: 'value',
+  filter: token => {
+    const usesDtcg = token.$type && token.$value;
+    return token[usesDtcg ? '$type' : 'type'] === PX_TO_REM_TOKEN_TYPE;
+  },
+  transform: token => {
+    const usesDtcg = token.$type && token.$value;
+    const baseFontSize = 16;
+    let value = token[usesDtcg ? '$value' : 'value'];
+    if (typeof value === 'string' && value.includes('px')) {
+      // Remove 'px' and convert the result to a number
+      value = parseFloat(value.replace('px', ''), 10);
```

Comment:
[parseFloat](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) only knows one parameter which need to be of type string.
And since it is parsing a string searching for a number, the `value.replace('px', '')` command is obsolete, because if `value` iquals `10px` it will return `10` as a unitless number anyway.

```suggestion
      value = parseFloat(value);
```